egs/babel_multilang/s5/local/nnet3/run_shared_ivector_extractor.sh

```diff
@@ -12,8 +12,8 @@ stage=4
 suffix=_sp
 feat_suffix=_hires # feat_suffix used in train_set for lda_mllt training.
 nnet3_affix=
-numLeavesMLLT=5500
-numGaussMLLT=90000
+numLeavesMLLT=2500
```

Comment:
IMO it would be better just not add more options and remove the MLLT training completely (i.e. replace it by PCA, as it's only the transforms we need). I think Dan Snyder has done enough experiments to show the difference is negligible if any at all.